packages/odyssey-react/src/components/Tabs/Tabs.tsx

```diff
@@ -196,26 +197,30 @@ const Tab = function TabsTab({
   });
 
   return (
-    <button
+    <Box
+      as="button"
       id={id}
       role="tab"
       tabIndex={selected ? 0 : -1}
       aria-controls={ariaControls}
       aria-selected={selected}
       className={componentClass}
       onClick={onClick}
+      color={false}
+      fontSize={false}
+      fontWeight={false}
```

Comment:
similar question to other PRs, how are you determining which properties need to be omitted here from the Box default styles?

Reply:
anything that is set by the component root element style needs to be opted out of or it gets overridden by Box and additionally cannot be themed.

The `.tab` style sets color, size, and weight
https://github.com/okta/odyssey/blob/d4bfae9c94c4906d4b67f085cb808939f9014126/packages/odyssey-react/src/components/Tabs/Tabs.module.scss#L18

Reply:
got it thanks that aligns with my understanding. Regarding theming I am going to create a JIRA to follow up on this batch of work so we can finalize the effort